easybuild/easyconfigs/m/MATLAB/MATLAB-2010b.eb

```diff
@@ -0,0 +1,23 @@
+name = 'MATLAB'
+version = '2010b'
+
+homepage = 'http://www.mathworks.com/products/matlab'
+description = """MATLAB is a high-level language and interactive environment 
+that enables you to perform computationally intensive tasks faster than with 
+traditional programming languages such as C, C++, and Fortran."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+sources = ['%s-%s.tar.gz'%(name.lower(), version)]
```

Comment:
for the record, a pure pythonic .iso extraction appears possible via: http://unix.stackexchange.com/a/164505